enhance(macros/CSSInfo): add "Animation type" link

mostafaroshdy1 opened the pull request:
<!--
  Thanks for taking the time to submit a pull request (PR)!
  Please provide enough information so that others can review your changes.

  The sections below are mandatory.
  If you don't follow this template, your PR will very likely be closed.

  Before submitting the PR, please make sure the following is done:
  1. Read the Community Participation Guidelines: https://www.mozilla.org/about/governance/policies/participation/
  2. Ensure that there is an open issue for the problem you're solving, or create it first: https://github.com/mdn/yari/issues/new/choose
  3. Fork the repository and create your branch from `main`.
  4. Run `yarn` in the repository root.
  5. Make sure to sign all your commits: https://docs.github.com/authentication/managing-commit-signature-verification/signing-commits
-->

## Summary

<!--
  Please reference the issue you're solving here.
  Example: Fixes #1234.
-->
This PR added the enchantment in this issue #8721 
### Problem

<!--
  Explain what problem the PR resolves in 1-3 sentences.
-->
The ''Animation type" string was not a hyperlink, so i made it an anchor tag referring to the related page

### Solution

<!--
  Explain how your PR solves this problem in 1-3 sentences.
  Please mention alternative solutions you have discarded, if any.
-->
- I have created a file in macros named AnimationType and added the localString as i found them in the official page [here](https://developer.mozilla.org/en-US/docs/Web/CSS/height#formal_definition) (in every other language)
- 

---

## Screenshots

<!--
  Did you change the user interface?
  If not, you can remove this section.
  If yes, please attach screenshots (or videos) of how the interface looks (or behaves) before and after your changes.
-->

### Before

![image](https://github.com/mdn/yari/assets/66712535/b3228591-1f57-409b-a01e-cd88c710d5d4)

### After
![image](https://github.com/mdn/yari/assets/66712535/c765e8c6-41d8-4767-b7ab-4d1a45f65e5b)

---

## How did you test this change?
Locally using ```yarn dev```

dependabot[bot] committed:
chore(deps): bump inquirer from 9.2.18 to 9.2.19 (#10940)

mostafaroshdy1 committed:
Added hyperlink to animation type

caugner commented on kumascript/macros/CSSInfo.ejs:
```diff
@@ -283,7 +283,7 @@ if (name === "preview-wiki-content") {
         properties = properties.concat({
             name: "animationType",
-            label: localize(localStrings, "animationType")
+            label: await template("animationType")
```
Let's avoid introducing a new macro here. Can you add the local string to this macro and build the link in line here?

mostafaroshdy1 replied:
done , you can check the latest commit , thanks

yin1999 committed:
enhance(macros/PreviousMenuNext): use the actual title of the document by default (#10812)

dependabot[bot] committed:
chore(deps): bump @webref/css from 6.12.7 to 6.12.8 (#10949)

dependabot[bot] committed:
chore(deps): bump openai from 4.36.0 to 4.37.1 (#10950)

dependabot[bot] committed:
chore(deps-dev): bump @swc/core from 1.4.15 to 1.4.16 (#10951)

dependabot[bot] committed:
chore(deps): bump web-specs from 3.7.1 to 3.8.0 (#10952)

caugner committed:
fix(playground): dispatch readystatechange + bubble DOMContentLoaded on document (#10946)

fix(playground): dispatch readystatechange event
fix(playground): dispatch DOMContentLoaded event on document + bubble

caugner committed:
feat(experiment): rewrite Web/API page titles (#10926)

Problem: We identified a drop in search engine impressions in April 2023, which coincides
with the Web/API retitle project (https://github.com/mdn/mdn/issues/284), and we cannot
rule out that there is a causal relationship.

Solution: Run an experiment with 500 randomly sampled Web/API pages (of 2719 affected),
partially reverting  the change to their `<title>` (but not to the `<h1>`) by essentially
replacing `Foo: bar property` with `Foo.bar property`.

caugner committed:
fix(experiment): replace test group for Web/API page titles (#10955)

The previous test group was not sampled from
the full population of affected Web/API pages.

github-actions[bot] committed:
chore(main): release 2.49.0 (#10933)

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

mostafaroshdy1 committed:
Changed the modification to inline

caugner commented on kumascript/macros/CSSInfo.ejs:
```diff
@@ -283,7 +283,7 @@ if (name === "preview-wiki-content") {
         properties = properties.concat({
             name: "animationType",
-            label: localize(localStrings, "animationType")
+            label: `<a href="/${env.locale}/docs/Web/CSS/CSS_animated_properties">${localize(localStrings, "animationType")}</a>`
```
Thank you, this makes me realize we should probably use `smartLink()` here. Can you try locally if this works?

```suggestion
            label: web.smartLink(`/${env.locale}/docs/Web/CSS/CSS_animated_properties`, null, localize(localStrings, "animationType"))
```

mostafaroshdy1 replied:
it worked just fine , thanks for the suggestion

mostafaroshdy1 committed:
Update kumascript/macros/CSSInfo.ejs

Co-authored-by: Claas Augner <495429+caugner@users.noreply.github.com>

mostafaroshdy1 committed:
Merge branch 'main' into add-hyperlink

mostafaroshdy1 commented:
@caugner 

I have updated the commits signature

caugner commented on yarn.lock:
```diff
@@ -11360,11 +11360,12 @@ open@^8.4.0:
     is-docker "^2.1.1"
     is-wsl "^2.2.0"
-
+    
```
```suggestion

```

caugner commented on yarn.lock:
```diff
@@ -11360,11 +11360,12 @@ open@^8.4.0:
   version "4.38.1"
   resolved "https://registry.yarnpkg.com/openai/-/openai-4.38.1.tgz#375ba1e2ff9fcd6434114bfeffa7a0ef35d74b78"
   integrity sha512-nmSKE9O2piuoh9+AgDqwGHojIFSxToQ2jJqwaxjbzz2ebdD5LYY9s+bMe25b18t4QEgvtgW70JfK8BU3xf5dRw==
+
```
```suggestion
```

caugner committed:
fix: revert yarn.lock changes

caugner commented on yarn.lock:
```diff
@@ -11360,7 +11360,6 @@ open@^8.4.0:
     define-lazy-prop "^2.0.0"
     is-docker "^2.1.1"
     is-wsl "^2.2.0"
```
```suggestion
    is-wsl "^2.2.0"

```

caugner committed:
fix: revert yarn.lock changes
